.jenkins/libcxx_tests.Jenkinsfile

```diff
@@ -127,7 +133,13 @@ try {
              "libcxx ACC1804 gcc Debug" :              { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'Debug') },
              "libcxx ACC1804 gcc Release" :            { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'Release') },
              "libcxx ACC1804 gcc RelWithDebInfo" :     { ACClibcxxTest(BIONIC_LABEL, 'gcc', 'RelWithDebinfo') }
+}catch(Exception e){
+    println "Caught global pipeline exception :" + e
+    GLOBAL_ERROR = e
```

Comment:
If we are setting Global Error and then throwing, is there a use of Global Error elsewhere?

Reply:
It's at used at the script scope level - it is only used again in the finally.